### PR TITLE
Feature/initial html for calculator

### PR DESCRIPTION
**Pull request by Piotr-code-tech**
I added input, radio buttons, button to calculate and output labels.

### CODE REVIEW

**Comment by pstudniak** [comment]
Commit messages don’t have to be merged with ’-’ or _ . This tyle applies only for branch names

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <h1>Learning App</h1>
+</head>
+<body>
+<form>  <!--I don't know what to do with metod attribut, it should be get or post now I don't write that -->
```
Comment (pstudniak) [inline src/index.html:8]: For now we are not sending form anywhere, you don’t have to specify HTTP method. We will add such functionality in the further development and probably use JavaScript to communicate with the backend instrad of doing that directly from HTML. You can read about HTTP methods and REST API for the future

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <div>
+        <label for="value">Please enter the value :</label>
```
Comment (pstudniak) [inline src/index.html:10]: Placeholder should contain tip for the user what kind of value is expected. It’s obvious that something should be written there. I’d suggest setting placeholder to ’value’. The same applies for labels.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <input type="number" name="value" id="value" min="1" placeholder="Write here"  required>
+    </div>
+    <div>
+        <button type="submit">Submit</button>
```
Comment (pstudniak) [inline src/index.html:14]: I like the idea with submit and reset buttons, but as for now we don’t need submit button. Please remove types attributes

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <button type="reset">Reset</button>
+    </div>
+    <b>Choose value with you want to calculate :</b>
```
Comment (pstudniak) [inline src/index.html:17]: <b> is deprecated. Use <strong> instead

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <h1>Learning App</h1>
+</head>
+<body>
```
Comment (pstudniak) [inline src/index.html:7]: We won’t be doing any calculatotions on the server now, you can swap form tag with some other semantically right block element

**Comment by pstudniak** [comment]
This pull request looks good overall. Please adjust to my comments and we can move foreward. Well done tbh

**Comment by Piotr-code-tech** [comment]
I didn't really know to which tag to change the form tag. I've nested everything from the form tag in div tag. The rest of the changes should be okay.

**Comment by pstudniak** [comment]
updated readme with next tasks. This PR is ready to be merged

**Comment by pstudniak** [comment]
please always answer to comments on PRs

File: src/index.html [hunk 6]
```diff
@@ -5,27 +5,29 @@
...
-        <button type="submit">Submit</button>
-        <button type="reset">Reset</button>
+        <button>Submit</button>
+        <button>Reset</button>
     </div>
-    <b>Choose value with you want to calculate :</b>
+    <strong>Which value do you want to calculate :</strong>
     <div>
         <label for="g_value">Gross value</label>
         <input type="radio" name="g/n_value" id="g_value" value="g_value">
```
[the remark this answers is not in the document]
Comment (Piotr-code-tech) [reply src/index.html:20]: ok, I change it

**Pull request merged**